lib/flagsmith/src/flagsmith-core.ts

```diff
@@ -622,16 +622,23 @@ const Flagsmith = class {
         this.updateEventStorage();
     }
 
-    getValue = (key:string) => {
+    getValue = (key:string, options?: GetValueOptions) => {
```

Comment:
If I'm not mistaken, this means we need to change https://github.com/Flagsmith/flagsmith-js-client/pull/147/files#diff-e2f04af8a0e801f4615f5e4e9d767731aebcb8644569cfdf168d8b21fb202ee7R114 and either IFlagsmithValue or the return type to ``IFlagsmithValue| Array<any> | object``

Reply:
I wonder if we can actually use generics here ``getValue<T= Array<any> | object> `` and ``GetValueOptions<T>``

Reply:
Missed the entry for `getValue` thanks for that :)
Typings should be now resolved thanks to the help of @dangrima90  

https://user-images.githubusercontent.com/2339134/193662430-ac4b2095-4ace-4aef-a449-44d736479b8a.mov